src/Backend/Modules/Blog/DataFixtures/LoadBlogComment.php

```diff
@@ -0,0 +1,111 @@
+<?php
+
+namespace Backend\Modules\Blog\DataFixtures;
+
+use Backend\Core\Language\Locale;
+use Backend\Modules\Blog\Domain\Category\Category;
+use Backend\Modules\Blog\Domain\Comment\Comment;
+use Common\Doctrine\Entity\Meta;
+use SpoonDatabase;
+use Symfony\Bundle\FrameworkBundle\Client;
+
+class LoadBlogComment
+{
+    /**
+     * @var Client
+     */
+    private $client;
+
+    public function __construct(Client $client)
+    {
+        $this->client = $client;
+    }
+
+    public function load(SpoonDatabase $database): void
+    {
+        $this->insertPost($database);
+
+        $comment = new Comment(
+            1,
+            Locale::fromString('en'),
+            'John Doe',
+            'john@example.com',
+            'This is just a short text, that represents a comment',
+            'comment',
+            'published',
+            'http://example.com',
+            null
+        );
+
+        $entityManager = $this->client->getContainer()->get('doctrine.orm.default_entity_manager');
+        $entityManager->persist($comment);
+        $entityManager->flush($comment);
+    }
+
+    private function insertPost(SpoonDatabase $database)
+    {
+        $meta = new Meta(
+            'Blogpost for functional tests', false,
```

Comment:
must be multiple lines

Reply:
fixed in 4c322d4485af9cf7eb3496e8b351045e61c6ff60